mono/metadata/boehm-gc.c

```diff
@@ -127,6 +121,7 @@ mono_gc_base_init (void)
 	mono_w32handle_init ();
 #endif
 
+#if JON
```

Comment:
Left over from debugging?

Reply:
Yes, this needs removed. All access in this block is using deprecated APIs and setting stackbottom is not needed.